File: 02_ProgrammingFundamentals/_03_Arrays/_01_Lecture/_01_DayOfWeek.java
package _02_ProgrammingFundamentals._03_Arrays._01_Lecture;

import java.util.Scanner;

public class _01_DayOfWeek {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Add variable
        String[] days = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday" ,"Saturday", "Sunday"};

        // Print result
        try {
            System.out.println(days[n - 1]);
        } catch (Exception e) {
            System.out.println("Invalid day!");
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_03_Arrays/_01_Lecture/_01_DayOfWeek/_01_DayOfWeek.cs
using System;

public class _01_DayOfWeek
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Add variable
        string[] days = { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };

        // Print result
        try
        {
            Console.WriteLine(days[n - 1]);
        }
        catch (Exception e)
        {
            Console.WriteLine("Invalid day!");
        }
    }
}
